index.js

```diff
@@ -1,41 +1,89 @@
 
 //------------------------ Question 1---------------------------
-//Write a function that ask the user for two numbers and return the product of them. But the challenge here is that you can't use the operator * (you can't make the product)
+//Write a function that ask the user for two numbers and return the product of them. 
+//But the challenge here is that you can't use the operator * (you can't make the product)
 //hint: multiplication is a sequence of sums
 
  //@param {number} number1
  //@param {number} number2
  //@return {number} product
-const multiplication = () => {
-  return
+
+let number1 = parseInt(prompt("Type a random number."));
+let number2 = parseInt(prompt("Type another random number."));
+
+ const multiplication = () => {
+  let product = 0;
+     for (let i = 1; i <= number2; i += 1){
+       product = product + number1;
+     }
+     return product
 }
 
-const answer1 = multiplication()
+const answer1 = multiplication (number1, number2)
 console.log('answer1', answer1)
 
 const htmlTarget = document.getElementById('a-1')
 htmlTarget.innerHTML = answer1
 
-
-
 //------------------------ Question 2 ---------------------------
-// Write a function that recieves a string as a parameter and evaluate each character of the string to determinate if the character is vowel or a consonant. you have to store each character on separates arrays, one for vowels and the other one for consonants.
+// Write a function that recieves a string as a parameter and evaluate each character of the string to determinate if the character is vowel or a consonant. 
+// You have to store each character on separates arrays, one for vowels and the other one for consonants.
 // after separating the characters concatenate both arrays. ask the user if wants the vowels first or consonants first in the final array.
 //hint:
 
 //@param {string}
 //@return {character} => array of characters
-vowelOrConsonant = () => {
-  return
+
+// My Answer
+let word = prompt("Type a random word.").toLowerCase();
+let vowels = [];
+let consonants = [];
+
+vowelOrConsonant = (word) => {
+  for(let i = 0; i < word.length; i += 1) {
+  if(word[i] === 'a'|| word[i] === 'e' || word[i] === 'i' ||  word[i] === 'o' || word[i] === 'u'){
```

Comment:
you could also store these "vowel" characters in an array to simplify your `if` statement:

```js
const vowels = ['a', 'e', 'i', 'o', 'u'];
if (vowels.includes(word[i]) {
  // ...
}
```